source/uwp/Renderer/lib/XamlHelpers.cpp

```diff
@@ -850,4 +857,33 @@ namespace AdaptiveCards::Rendering::Uwp::XamlHelpers
 
         return {lineColor, spacing, lineThickness};
     }
+
+    winrt::Image XamlHelpers::RenderImageFromDataUri(winrt::Uri const& imageUrl)
+    {
+        winrt::Image image{};
+        winrt::BitmapImage bitmapImage{};
+        bitmapImage.CreateOptions(winrt::BitmapCreateOptions::IgnoreImageCache);
+
+        // Decode base 64 string
+        winrt::hstring dataPath = imageUrl.Path();
```

Comment:
nit: is it possible to not create local variable?
AdaptiveBase64Util::ExtractDataFromUri(HStringToUTF8(imageUrl.path()))